.env.example

```diff
@@ -1,4 +1,10 @@
-LOG_LEVEL= "info"
-RPC_URL = http://localhost:8545
-OUTPUT_TYPE = "graylog"
-ACCESS_TOKEN = ""
\ No newline at end of file
+LOG_LEVEL= 'debug'
+SOCKET_IO_PORT = 3030
+RPC_URL = "https://nd.neufund.org:2096"
+OUTPUT_TYPE = "terminal"
+ACCESS_TOKEN = ""
+SAVE_STATE = "true"
```

Comment:
I still don't understand the meaning of string including "true" is the double quotes are mistake, or it mean something?

second thing why did you move it into the .env file?

Reply:
Now with this change everything can be an env variable, this is better for system admins to choose what settings are bet for them